src/styles/text.tsx

```diff
@@ -34,12 +34,17 @@ export const CenteredH3 = styled(H3)`
   align-self: center;
 `;
 
-export const H4 = styled.h4<{ $color?: string }>`
+export const H4 = styled.h4<{ $color?: string; $required?: boolean }>`
   ${openSans.style}
   font-size: 1rem;
   font-weight: 600;
   color: ${props => props.$color || COLORS.greyDarker};
   margin: 0;
+  ${({ $required }) =>
+    $required &&
+    `&::after {
+  content: ' *';
+  color: ${COLORS.redMid}}`};
```

Comment:
not sure if this makes sense for an `H4`, let's put the `required` prop in the respective `InputLabel` instead
or we can bring `InputLabel` into here since it is reused quite often

Reply:
mmm yeah i was also thinking that i was a little weird for H4; good idea about inputlabel. lemme do that rn